README.md

```diff
@@ -1,13 +1,36 @@
 # collect_stata
 
-Accumulate data from stata files and write it into an open format 
+The tool reads dta-files, calculate some univariate statistics for each variable 
+and writes out a json-file with these calculations and the metadata.
```

Comment:
What is "metadata" in this case?

Reply:
extracted from stata - the description of the given data